src/components/App/App.component.js

```diff
@@ -1,5 +1,5 @@
 // @flow
```

Comment:
I agree about the EventCaptureForm, but I would like to keep the components folder. 1. Because it separates the React components from the other stuff 2. The capture-core is also structured this way. 

Let's leave it for another issue though (trying to stay on point with the issue at hand here).